README.md

```diff
@@ -12,8 +12,12 @@ to manage **countless** potential and current clients.<br>
 Many financial advisors have **trouble scheduling** for meet-ups due to
 information overload of their clients.<br>
 
-**FABook** is optimized for a **one-stop storage** to store and remind you on
-your client's information.
+**FABook** is optimized for a **one-stop storage solution** to store and remind them on
```

Comment:
Are we using 'them' for personal pronouns instead of 'you'?

Reply:
For this i think its a readme so i think dont have to be "talking to the user" this is like generic what it does. make sense?

Reply:
yup makes sense since is readme